src/RSSFeed.php

```diff
@@ -736,7 +736,7 @@ public static function showListForCentral(bool $personal = true, bool $display =
             $criteria['WHERE']["$table.users_id"] = $users_id;
             $criteria['WHERE']["$table.is_active"] = 1;
 
-            $titre = "<a href='" . htmlspecialchars($CFG_GLPI["root_doc"] . "/front/rssfeed.php") . "'>" .
+            $titre = "<a href='" . RSSFeed::getSearchURL() . "'>" .
```

Comment:
Please preserve the usage of encoding functions, even if the URL is not supposed to contain any special char. Some specific URLs may contains some user input and it is easier, for developpers, to consider every URL as potentially unsafe rather than having to verify, for each, if the special chars encoding can be missing.
```suggestion
            $titre = "<a href='" . htmlspecialchars(RSSFeed::getSearchURL()) . "'>" .
```